@AH-64D Apache Longbow/Addons/fza_ah64_weapons/functions/pylon/fn_PylonSwapper.sqf

```diff
@@ -0,0 +1,67 @@
+/* ----------------------------------------------------------------------------
+Function: fza_weapons_fnc_PylonSwapper
+
+Description:
+	swaps the hellfire and rockets between the pylons
+	
+Parameters:
+	_heli - The helicopter to get information from [Unit].
+
+Returns:
+	...
+
+Examples:
+	...
+
+DEBUG:
+	_heli = vehicle player; 
+    _wassing = "rockets"; 
+    _turret = -1; 
+    _PylonInfo = getAllPylonsInfo vehicle player; 
+    if (vehicle player turretLocal [-1]) then {_turret = 0;}; 
+    [_wassing,_PylonInfo,_heli] remoteExec ["fza_weapons_fnc_pylonswapper", vehicle player turretUnit [_turret]]; 
+    [_wassing,_PylonInfo,_heli] call fza_weapons_fnc_pylonswapper;
+
+
+Author:
+	Rosd6(Dryden)
+---------------------------------------------------------------------------- */
+params ["_wasselect","_PylonInfo","_heli"];
```

Comment:
capitalisation should be `params ["_wasSelect","_pylonInfo","_heli"];

Generally the standard is to put `_heli` first, please can you swap them around?